go/plugins/vertexai/vertexai.go

```diff
@@ -169,6 +172,9 @@ func defineModel(name string, caps ai.ModelCapabilities) ai.Model {
 		input *ai.ModelRequest,
 		cb func(context.Context, *ai.ModelResponseChunk) error,
 	) (*ai.ModelResponse, error) {
+		if strings.HasPrefix(name, "imagen") {
```

Comment:
This code block is auto-generated/copied from googleai.go so it's going to get overwritten. Can you figure out a way around it?

Reply:
I just put outside of the code copied